src/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>The German Human Genome-Phenome Archive</title>
+    <title>GHGA Archive</title>
```

Comment:
The whole product is the "GHGA Archive", but the web client is called "GHGA Data Portal". This also makes it clear that this is not the GHGA homepage. And the "A" in GHGA already stands for "Archive" so "GHGA Archive" is actually an awkward duplication.

Reply:
That's also why the URL was chosen to be data.ghga.de and not archive.ghga.de.